lighthouse-cli/bin.ts

```diff
@@ -330,8 +330,14 @@ export async function runLighthouse(url: string,
     const chromeLauncher = await getDebuggableChrome(flags)
     const results = await lighthouse(url, flags, config);
 
+    // Filter out artifacts except for screenshots in traces to minimize report size.
+    var traces = results.artifacts.traces;
+    for (var pass in traces)
+      traces[pass]['traceEvents'] = traces[pass]['traceEvents'].filter(e => e['cat'] === 'disabled-by-default-devtools.screenshot');
```

Comment:
oh this is actually orthogonal to speedline perf IMO, we should definitely do that too, but even if we didn't we can still leverage the SSIM to only return thumbnails that are substantively different